src/rules/mongodb.js

```diff
@@ -7,10 +7,20 @@
  *
  */
 
-function (user, context, callback) {
-  const connection_string = configuration.MONGO_CONNECTION_STRING;
-  mongo(connection_string, function (db) {
+function enrichProfileWithMongo(user, context, callback) {
+  const MongoClient = require('mongodb@3.1.4').MongoClient;
+  const mongoUrl = configuration.MONGO_CONNECTION_STRING;
+
+  if (!mongoUrl || /^mongo:\/\//.exec(mongoUrl)){
```

Comment:
should we add a success and fail test for the regex?

Reply:
Yep, sounds good. I copied this from the mongo wrapper script that was baked into auth0-rulez-api, but we should add definitely add tests for this as well.

Reply:
Or, we could just remove this check entirely since the mongo library does its own validation of the URL. Does that work for you instead?

Reply:
I went ahead and remove the check since the mongo client does one as well.